add roles feature to manifest

dennwc opened the pull request:
Signed-off-by: Denys Smirnov <denys@sourced.tech>

juanjux commented:
I was thinking... one of the `uast` or `roles` is redundant. If the driver works as a minimal viable version, it'll already generate an UAST (unannotated) so either you have roles, that you could mark with `uast` or `roles`, or not but you can't have not uast.

dennwc commented:
I was also thinking about this. Current list of features is actually a level (higher level implies lower is supported), but it might not be the case in the long run.

I was assuming that this question will be raised in https://github.com/bblfsh/sdk/pull/233, where I introduced these features. Now it's a bit late - I already updated all the drivers according to current approach. I can change an SDK to add lower level features automatically, but it's not a part of this PR and will require to update all other drivers again.

juanjux commented:
Yes, didn't notice at that time, whatever, we'll need to do a lot of changes soon with the UAST2 changes so we can fix that then.
